dataeng/resources/enrollment-validation-events.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+END_DATE=$(date +%Y-%m-%d -d "$TO_DATE")
+START_DATE=$(date +%Y-%m-%d -d "$FROM_DATE")
+INTERVAL=$START_DATE-$END_DATE
+
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY + 1 ))
+fi
+env|sort
+
+${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
+  EnrollmentValidationWorkflow --local-scheduler \
+  --output-root $OUTPUT_ROOT/$INTERVAL \
+  --interval $INTERVAL \
+  --credentials $CREDENTIALS \
+  --n-reduce-tasks $NUM_REDUCE_TASKS
+
+pip install awscli
+aws s3 ls $OUTPUT_ROOT/$INTERVAL/ | grep "synthetic_enroll"  || true
+
+MAX_FILE_SIZE=`aws s3 ls $OUTPUT_ROOT/$INTERVAL/ | cut -b20-30 | grep -v PRE | sort -nr | head -1`
+if [ -z "$MAX_FILE_SIZE" ] && [ "$FILE_THRESHOLD" -gt "0" ]; then :
```

Comment:
@brianhw does this look reasonable ?

Reply:
I *think* it does the right thing, but I think it would be clearer if it didn't check for file threshold.    That is, I think it could just be:  if [ -z "$MAX_FILE_SIZE" ]; then :   which should be true if there are no synthetic events.  It shouldn't matter what the FILE_THRESHOLD value is.

Reply:
I've combined it into a single if else.